### ISSUE TITLE
UI: Rounded Edges

### ISSUE THREAD

**Issue by peterklingelhofer** [resonatecoop/beam#193]
Rounded button edges better for mental health, and those with low vision:
 - https://sandykozar.decoratingden.com/psychology-interior-design/
 - https://blog.bitsrc.io/rounded-corners-over-sharp-corners-f71008a51afb
 - http://macknik.neuralcorrelate.com/pdf/articles/troncoso_et_al_Art_and_Perception_chapter.pdf
 - https://youtu.be/A_u2WFTfbcg
   - research indicates amygdala associated with fear and anxiety lights up when people look at angular objects as opposed to round ones round ones

**Comment by simonv3** [comment]
https://github.com/resonatecoop/beam/blob/main/src/components/common/Button.tsx#L63

They are currently rounded. I guess you're asking for more rounded?

**Comment by peterklingelhofer** [comment]
Yeah the buttons are good as is. Was more referring to pop-up modal windows. Should have specified!

![image](https://user-images.githubusercontent.com/60944077/202880716-f574435b-7b5f-4238-b595-d0b28b234ac4.jpeg)
